tslib/src/functions/check-marker/index.ts

```diff
@@ -1,34 +1,34 @@
-/* eslint-disable import/first */
-const functionName = 'check-marker'
-import aiHelper from '../../azure/app-insights'
-// load early to enable tracking
-aiHelper.startInsightsIfConfigured(functionName)
-
-import { AzureFunction, Context } from '@azure/functions'
-import { performance } from 'perf_hooks'
-import * as V1 from './check-marker.v1'
-import { ICheckMarkerFunctionBindings } from './models'
-import { MarkCheckMessageV1 } from '../../schemas/models'
-const marker = new V1.CheckMarkerV1()
-
-const serviceBusQueueTrigger: AzureFunction = async function (context: Context, markCheckMessage: MarkCheckMessageV1): Promise<void> {
-  const start = performance.now()
-  const version = markCheckMessage.version
-  context.log.info(`${functionName}: version:${version} message received for checkCode ${markCheckMessage.checkCode}`)
-  try {
-    if (version !== 1) {
-      throw new Error(`Message schema version ${version} unsupported`)
-    }
-    await marker.mark(context.bindings as ICheckMarkerFunctionBindings, context.log)
-  } catch (error) {
-    context.log.error(`${functionName}: ERROR: ${error.message}`)
-    throw error
-  }
-
-  const end = performance.now()
-  const durationInMilliseconds = end - start
-  const timeStamp = new Date().toISOString()
-  context.log(`${functionName}: ${timeStamp} run complete: ${durationInMilliseconds} ms`)
-}
-
-export default serviceBusQueueTrigger
+/* eslint-disable import/first */
```

Comment:
is this just re-formatted?

Reply:
Yes, just a whitespace change - line endings were changed to \n